packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -1,45 +1,374 @@
+            <!-- Logo image -->
+            <restaurant-logo
+                v-if="logoUrl"
+                :class="$style['c-restaurantCard-logo']"
+                :logo-url="logoUrl" />
+            <!-- Tags inside image container -->
+            <component
+                :is="errorBoundary"
+                v-if="hasImageTags"
+                :tier="3">
+                <restaurant-tags
+                    :class="$style['c-restaurantCard-imageTags']"
+                    test-id-position="main-image"
+                    :tags="tags.imageTags" />
+            </component>
+
+        </restaurant-image>
+
+        <!-- primary content -->
+        <div :class="$style['c-restaurantCard-content']">
+            <!-- Restaurant Name -->
+            <h3
+                :class="$style['c-restaurantCard-name']"
+                data-test-id="restaurant_name"
+                data-search-name>
+                {{ name }}
+            </h3>
+
+            <!-- Cuisines -->
+            <component
+                :is="errorBoundary"
+                v-if="cuisines.length > 0"
+                :tier="3">
+                <restaurant-cuisines
+                    data-test-id="restaurant-cuisines"
+                    :cuisines="cuisines" />
+            </component>
+
+            <!-- Premier Icon -->
+            <legend-icon
+                v-if="isPremier"
+                :class="[$style['c-restaurantCard-premier']]"
+                data-test-id="premier-icon" />
+
+            <!-- New label -->
+            <!--
+                TODO - we want to translate this within the component using i18n.
+                For now we'll just need to pass down a translated string from the consuming site
+            -->
+            <restaurant-tag
+                v-if="newTagText"
+                :is-large="true"
+                :is-uppercase="true"
+                :text="newTagText"
+                color-scheme="positive" />
+
+            <!-- Ratings -->
+            <component
+                :is="errorBoundary"
+                v-if="rating"
+                :tier="3">
+                <restaurant-rating
+                    data-test-id="restaurant-rating"
+                    v-bind="rating" />
+            </component>
+
+            <component
+                :is="errorBoundary"
+                v-if="availability"
+                :tier="3">
+                <restaurant-availability
+                    v-bind="availability"
+                    data-test-id="restaurant-availability" />
+            </component>
+
+            <!-- Offline Icon -->
+
+            <!-- Meta Items List -->
+            <component
+                :is="errorBoundary"
+                v-if="displayDeliveryTimeMeta"
+                :tier="3">
+                <delivery-time-meta
+                    v-bind="deliveryTimeData"
+                    data-test-id="restaurant-delivery-time-meta" />
+            </component>
```

Comment:
Might want to change this to `data-test-id="delivery-time-meta"` aso it matches the component name and follows your other component test ids